src/blue-theme.scss

```diff
@@ -0,0 +1,6 @@
+@charset "utf-8";
+$democracy-blue: #2e4272;
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)